CHANGELOG.md

```diff
@@ -4,8 +4,13 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.0] - 2020-05-22
+### Added
+- Add if_truthy formatter
```

Comment:
Can this be a little more informative as to what the addition is?

Reply:
It would probably be good to also start documenting what formatters exist as part of the `README.md`. Could you start building out a table under the `Usage` section that lists this one out and has a comment to backfil?

Reply:
Updated CHANGELOG and README here: 178d1aa3d0ac6875b91033c39944c0c91154846a